pages/installation/os/ubuntu.rst

```diff
@@ -19,9 +19,12 @@ Taking a minimal server setup as base will need this additional packages::
 MongoDB
 -------
 
-The Version included in Ubuntu 16.04 LTS can be used together with Graylog 2.4.x::
+The official MongoDB repository provides the most up-to-date version and is the recommended way of installing MongoDB for Graylog 2.4.x on Ubuntu 16.04 LTS::
 
-    $ sudo apt-get install mongodb-server
+    $ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2930ADAE8CAF5059EE73BB4B58712A2291FA4AD5
+    $ echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu `lsb_release -cs`/mongodb-org/3.6 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.6.list
```

Comment:
Please don't overdo it here with `lsb_release`, which might not work as expected on minimal installations or custom Ubuntu images.

Using "xenial" here is fine, because we explicitly target Ubuntu 16.04 LTS with this guide.